hs/app/reachpc/ReachPC/Config.hs

```diff
@@ -63,12 +72,92 @@ data RchToml = RchToml
   , rtml_organization :: !(Maybe String)
   } deriving (Show)
 
+emptyRchToml :: RchToml
+emptyRchToml = RchToml Nothing Nothing Nothing Nothing Nothing
+
+-- helper for interactiveConfigCreate
+data PromptAnswer = Unset | Unchanged | Str String
+
+-- Code for the `reach config` subcommand
+interactiveConfigCreate :: IO ()
+interactiveConfigCreate = do
+  which <- askUserEnum "Which config file do you want to edit?" ["Global", "Project"]
+  let global = which == "Global"
+  (rchTmlPath, rchTml) <- if global then readOrEmptyGlobalReachToml else readOrErrorProjectReachToml
+  let showCurrent field = "Currently " <> maybe "unset" (\f -> "\"" <> show f <> "\"") (field rchTml)
+
+  project <- case global of
+              True -> return Nothing
+              False -> askUserString ("\nWhat is the remote name of the project? " <> showCurrent rtml_project) "Project name: "
+                       <&> \case
+                         Unset -> Nothing
+                         Unchanged -> rtml_project rchTml
+                         Str s -> Just s
+  organization <- askUserString ("\nWhat is your organization? " <> showCurrent rtml_organization) "Organization: "
+                  <&> \case
+                    Unset -> Nothing
+                    Unchanged -> rtml_organization rchTml
+                    Str s -> Just s
+  cloudOrLocal <- askUserEnum' ("\nDo you want to use Reach Cloud or Reach Local? " <> showCurrent rtml_cloudOrLocal) (showEnum @CloudOrLocal)
+                  <&> \case
+                    Unset -> Nothing
+                    Unchanged -> rtml_cloudOrLocal rchTml
+                    Str s -> Just $ readCloudOrLocal s
+  connector <- askUserEnum' ("\nWhat connector do you want to use? " <> showCurrent rtml_connector) (showEnum @Connector)
+               <&> \case
+                 Unset -> Nothing
+                 Unchanged -> rtml_connector rchTml
+                 Str s -> Just $ readConnector s
+  forwardEnvVars <- askUserString ("\nWhat environment variables do you want to forward? " <> showCurrent rtml_cloudOrLocal) "Env vars (comma separated list, e.g. VAR1,VAR2,VAR3): "
+                    <&> \case
+                      Unset -> Nothing
+                      Unchanged -> rtml_forwardEnvVars rchTml
+                      Str s -> Just . map (dropWhileEnd isSpace . dropWhile isSpace) . splitOn "," $ s
+  let newRchTml = RchToml
+        { rtml_project = project
+        , rtml_organization = organization
+        , rtml_cloudOrLocal = cloudOrLocal
+        , rtml_connector = connector
+        , rtml_forwardEnvVars = forwardEnvVars
+        }
+
+
```

Comment:
Yuck

Reply:
<img src="https://c.tenor.com/cOtdrAgJPO8AAAAd/stanley-hudson-the-office.gif" width=20% height=20%> how did you even see the separation? good eye